misc/cohen3.conf.example

```diff
@@ -80,8 +80,8 @@ serverport = 30020                       # if not specified or set to 0
     #[[IRadioStore]]
     #    name = iRadio
 
-    [[FlickrStore]]
-        name = Flickr Images
+    #[[FlickrStore]]
+    #    name = Flickr Images
```

Comment:
Well, it's easy to find the package icons, but it'd require a line of code to resolve the python package location.  I was thinking it best to leave it all up to the user, but if you really want it, it can be done.